### DISCUSSION TITLE
Github issues now supported by the failure exporter

### DISCUSSION THREAD

**Discussion by weshayutin** [dora-metrics/pelorus#491] [Show and tell]
Check out the Github Issues failure exporter and please provide your feedback!
https://github.com/konveyor/pelorus/blob/master/docs/Configuration.md#instance-config-github

**Comment by mateusoliveira43** [comment]
Closing as we have this documented in the read the docs website 
- https://pelorus.readthedocs.io/en/latest/GettingStarted/Overview/#supported-providers
- https://pelorus.readthedocs.io/en/latest/UpstreamSupport/#failure-exporter
